webapp/src/js/bootstrapper/index.js

```diff
@@ -262,27 +267,28 @@
             if (!shouldPurgeMeta) {
               return;
             }
-
+            window.startupTimes.purgeMetaStarted = performance.now();
             setUiStatus('PURGE_META');
             return purger.purgeMeta(localMetaDb);
           })
-          .catch(err => console.error('Error attempting to purge meta', err));
+          .catch(error => console.error('Error attempting to purge meta', error))
+          .then(() => window.startupTimes.purgeMetaEnded = performance.now());
```

Comment:
Thinking of adding this here after the catch as it's completely finished.

Reply:
This works. What do you have against `err`? :D

Reply:
It's an actual word: https://www.merriam-webster.com/dictionary/err :D

Reply:
Noooo lol, I can actually say "erred" or "erring" 🤣
I'll revert the change

Reply:
Done